semgrep/semgrep/spacegrep.py

```diff
@@ -88,13 +87,6 @@ def run_spacegrep(
                     output_json["matches"] = _patch_id(
                         pattern, output_json.get("matches", [])
                     )
-                    # dedent lines
```

Comment:
To be consistent with other parts of the codebase, let's not dedent here. We don't do this for regular patterns, e.g.

```
$ cat /tmp/test.py
if True:
    print(1 == 1)  # This is a comment
```

```
$ python -m semgrep --json -e '$X == $X' -l python /tmp/test.py
ran 1 rules on 1 files: 1 findings
{
    "results": [
        {
            "check_id": "-",
            "path": "/tmp/test.py",
            ...
            "extra": {
                ...
                "lines": "    print(1 == 1)  # This is a comment"
            }
        }
    ],
    "errors": []
}
```

Note the whitespace in `lines` for a regular pattern.